books/free-programming-books-ta.md

```diff
@@ -98,6 +105,11 @@
 * [எளிய தமிழில் Agile/Scrum](https://freetamilebooks.com/ebooks/learn-agine-scrum-in-tamil) - த.சீனிவாசன் (PDF)
 
 
+### Software Testing
+
+* [எளிய தமிழில் Software Testing](https://freetamilebooks.com/ebooks/eliya_tamilil_software_testing/) - கி.முத்துராமலிங்கம் (PDF)
```

Comment:
Don't invent titles! Use the title given by the resource.

Reply:
@eshellman Please review it now.